apps/rstudio/template/script.sh.erb

```diff
@@ -44,11 +44,14 @@ export RSESSION_LOG_FILE="$OOD_SESSION_STAGED_ROOT/rsession.log"
 <%= context.global_prerun.gsub("\r", "") %>
 <%- end -%>
 
+# System preferences at $RSTUDIO_CONFIG_DIR/rstudio-prefs.json (default location = /etc/rstudio)
+# User preferences have priority over system preferences
+export RSTUDIO_CONFIG_DIR="$OOD_SESSION_STAGED_ROOT/config_dir"
+
 CLEAN_WORKSPACE=<%= context.clean_workspace %>
-if [[ "${CLEAN_WORKSPACE}" == "yes" ]]; then
-    # System preferences at $RSTUDIO_CONFIG_DIR/rstudio-prefs.json (default location = /etc/rstudio)
-    # User preferences have priority
-    export RSTUDIO_CONFIG_DIR="$OOD_SESSION_STAGED_ROOT/clean_workspace"
+if [[ "${CLEAN_WORKSPACE}" == "no" ]]; then
+    # default value for "load_workspace" is true
+    sed -i '/load_workspace/d' $RSTUDIO_CONFIG_DIR/rstudio-prefs.json
```

Comment:
this is a bit dangerous, it depends on the default and deleting a line in a json file can lead to broken syntax. I suggest to use some kind of template in `rstudio-prefs.json`, for instance:
```
{
    "load_workspace": %%LOAD_WORKSPACE%%,
    "graphics_backend": "ragg"
}
```
and always `sed` that magic string to either true or false.

Reply:
or use the proper tool for json:
```
jq 'del(.load_workspace)'
```

Reply:
then we add it to the RL8 too